single_pages/dashboard/store/settings/tax.php

```diff
@@ -1,105 +1,122 @@
                         <div class="col-sm-7">
-                        <div class="form-horizontal">
-                            <p><strong><?= t("Matches...")?></strong></p>
-                            <div class="form-group">
-                                <label for="taxCountry" class="col-sm-5 control-label"><?= t("Country")?> <small class="text-muted"><?= t("Required")?></small></label>
-                                <div class="col-sm-7">    
-                                    <?php $country = $taxRate->getTaxCountry(); ?>
-                                    <?= $form->select('taxCountry',$countries,$country?$country:'US',array("onchange"=>"updateTaxStates()")); ?>
-                                </div>
-                            </div>
-                            
-                            
-                            <div class="form-group">
-                                <label for="taxState" class="col-sm-5 control-label"><?= t("Region")?> <small class="text-muted"><?= t("Optional")?></small></label>
-                                <div class="col-sm-7"> 
-                                    <?php $state = $taxRate->getTaxState(); ?>
-                                    <?= $form->select('taxState',$states,$state?$state:""); ?>
-                                    <?= $form->hidden("savedTaxState",$state); ?>
-                                </div>
-                            </div>
-        
-                            <div class="form-group">
-                                <label for="taxState" class="col-sm-5 control-label"><?= t("City")?> <small class="text-muted"><?= t("Optional")?></small></label>
-                                <div class="col-sm-7"> 
-                                    <?= $form->text('taxCity',$taxRate->getTaxCity());?>
-                                </div>
-                            </div>
+                            <?php $state = $taxRate->getTaxState(); ?>
+                            <?= $form->select('taxState',$states,$state?$state:""); ?>
+                            <?= $form->hidden("savedTaxState",$state); ?>
                         </div>
+                    </div>
+
+                    <div class="form-group">
+                        <label for="taxState" class="col-sm-5 control-label"><?= t("City")?> <small class="text-muted"><?= t("Optional")?></small></label>
+                        <div class="col-sm-7">
+                            <?= $form->text('taxCity',$taxRate->getTaxCity());?>
                         </div>
                     </div>
-            
+                </div>
+                </div>
+            </div>
+
+            <h4><?= t("VAT Number Options")?></h4>
+            <div class="row">
+
+                <div class="col-sm-12">
+
+                    <?php if (Config::get('community_store.vat_number')) { ?>
```

Comment:
If the VAT option isn't shown, the VAT Number Options label is still shown - so this check should include line 90